nf_core/pipeline-template/bin/scrape_software_versions.py

```diff
@@ -11,7 +11,7 @@
         software = "{{ name }}"
 
     with open(version_file) as fin:
-        version = fin.read().strip()
+        version = fin.read().strip().replace("\n", ", ")
```

Comment:
Sorry, now it's me being slow... I thought we get multiple files with one version each, instead of one file with one version per line?

Reply:
That's what `collectFile` was for. It merges all the different versions into one file to help with staging the file for the `GET_SOFTWARE_VERSIONS` process.

Reply:
ah, got it!